util/test/Test/Pos/Util/Tripping.hs

```diff
@@ -0,0 +1,83 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Test.Pos.Util.Tripping where
+
+import           Data.Aeson (FromJSON, ToJSON, decode, encode)
+import           Data.Text.Internal.Builder (fromText, toLazyText)
+import           Formatting.Buildable (Buildable (..))
+import           Hedgehog (Group, MonadTest, discoverPrefix, success, tripping)
+import           Hedgehog.Internal.Property (Diff (..), failWith)
+import           Hedgehog.Internal.Show (valueDiff)
+import           Hedgehog.Internal.TH (TExpQ)
+import qualified Prelude
+import           Text.Show.Pretty (Value (..), parseValue)
+
+import           Universum
+
+--------------------------------------------------------------------------------
```

Comment:
What's this separator for? If it's a logical section of code, perhaps we could document it a bit more?

Reply:
This was potentially a copy paste, but it doesn't serve any purpose IMO. I'll remove it.